contracts/src/CroBridge.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity ^0.6.11;
```

Comment:
Should we remove this if it's not used? 
if it's for integration testing, we could also move it to `integration_tests/contracts/`.